dev-docs/source/Testing/MSlice/MSliceTestGuide.rst

```diff
@@ -71,19 +134,124 @@ Taking Cuts
 #. Click ``Display`` in the ``Slice`` tab without changing the default values
 #. On the slice plot, select ``Interactive Cuts``
 #. Use the cursor to select a rectangular region in the slice plot. A second window with a cut plot should open.
+#. Check that the menu item ``Intensity`` is disabled as well as the item ``Recoil lines`` within the menu item ``Information`` in the new plot window
+#. Check that the ``File`` menu only has one menu item, ``Close``
 #. Change the rectangle by changing its size or dragging it to a different area of the slice plot. The cut plot should update accordingly.
+#. Click on ``Save Cut to Workspace`` and check the ``MD Histo`` tab of the Workspace Manager to verify that the new workspace was added
+#. Click on Flip Integration Axis. The y axis label changes from ``Energy Transfer (meV)`` to :math:`|Q| (\mathrm{\AA}^{-1})` or vice versa, depending on the initial label.
```

Comment:
I can't find Flip Integration Axis as an option. Is it in a menu somewhere? 

Also there is a formatting issue with the :math: part of this line.

Reply:
I think the formatting issue is caused by a build setting